R/fit_best.R

```diff
@@ -84,6 +90,7 @@ fit_best.tune_results <- function(x,
                                   metric = NULL,
                                   parameters = NULL,
                                   verbose = FALSE,
+                                  use_validation_set = NULL,
```

Comment:
Let's call this `train_with_validation` instead?

Reply:
after the team discussion we went with `add_validation_set`